lib/imagecanvas.h

```diff
@@ -73,8 +73,8 @@ class SLATE_EXPORT ImageCanvas : public QQuickItem
     Q_PROPERTY(QColor rulerBackgroundColour READ rulerBackgroundColour WRITE setRulerBackgroundColour)
     Q_PROPERTY(int cursorX READ cursorX NOTIFY cursorXChanged)
     Q_PROPERTY(int cursorY READ cursorY NOTIFY cursorYChanged)
-    Q_PROPERTY(int cursorSceneX READ cursorSceneX NOTIFY cursorSceneXChanged)
-    Q_PROPERTY(int cursorSceneY READ cursorSceneY NOTIFY cursorSceneYChanged)
```

Comment:
Is it necessary to remove these? Are they not used anywhere?

If they can be removed, we should probably remove the getters, too.